driver.go

```diff
@@ -12,49 +12,80 @@ import (
 	"github.com/alexbrainman/odbc/api"
 )
 
+const (
+	DriverPoolModeNone  DriverPoolMode = 0
```

Comment:
Let me explain, please.

The point here is to provide to the application a way to check/control if the underlying ODBC Driver Manager supports connection pooling, at least partially (no relaxed connection pooling), or not at all. In this way, the application can take an appropriate action about that (e.g. report an error, either accept or reject to use a db instance that partially supports connection pooling).

I agree, these constants are not strictly necessary for the users, however, in my opinion, we should at least keep them as private constants, in order to offer the opportunity to check the connection pooling status (through the functions Driver.IsPooling() and Driver.IsFullPooling()), for managing the application control flow according to that.

This is just a useful plus.
Right, it is not going to be useful for the greater part of the users. 
But for some users it might be important.
So, why remove it? ;) 

For instance, take the golang library "math/big".
It is not useful for many users, but it is very important for some users.

Reply:
I am not convinced that adding new exported functionality is useful here. How will you use these in your code?

And IsPooling() and IsFullPooling are terrible names. How is IsPooling not full? What do we do if we discover another shade of pooling in the future? The only sensible things to do here is to use original Microsoft names - but these names aren't better.

I suggest we keep thing private until someone really needs them.